synapse/handlers/room_list.py

```diff
@@ -313,7 +313,9 @@ async def generate_room_entry(
 
         # Return whether this room is open to federation users or not
         create_event = current_state[EventTypes.Create, ""]
-        result["m.federate"] = create_event.content.get("m.federate", True)
+        result["m.federate"] = create_event.content.get(
```

Comment:
Is this `"m.federate"` intentionally left in?

Reply:
I left it in because it describes the API shape (as opposed to event content). They *are* the same (and I assume they're the same because it is the name of the event field)... Do you think I should update it? I could go either way.

Reply:
I've got no strong opinion on this one. It's up to you :)